pkg/dlna/cds.go

```diff
@@ -333,8 +334,16 @@ func (me *contentDirectoryService) Handle(action string, argsXML []byte, r *http
 				if err != nil {
 					return nil, err
 				}
+
+				// http://upnp.org/specs/av/UPnP-av-ContentDirectory-v1-Service.pdf
+				// maximum update ID is 2**32, then rolls back to 0
+				maxUpdateID := int64(math.Pow(float64(2), float64(32)))
```

Comment:
using math and power to get 2^32 is not necessary
a simple left binary shift is enough
```golang
var maxUpdateID int64 = 1 << 32
```
[playground snippet](https://play.golang.org/p/eDsz5vkGxty)